src/mocks/handlers.js

```diff
@@ -1,7 +1,11 @@
 import { rest } from 'msw';
-export const handlers = [
-	rest.get("/",(req, res, ctx) => {
-		return res(ctx.json({"welcomeMessage" : "Hello World!"}))
-	})
 
-]
\ No newline at end of file
+export const handlers = [
+  rest.get('/https://pokeapi.co/api/v2/pokemon/', (req, res, ctx) => {
+    return res(
+      ctx.json({
+        welcomeMessage: 'Hello World!',
```

Comment:
This should be updated to return the actual data for a pokemon. If you hit the end point in the browser it will give you the data as a big JSON object. Past that here instead of the welcome message.